gwpy/table/io/root.py

```diff
@@ -75,47 +75,38 @@ def table_from_root(source, treename=None, **kwargs):
             return Table(trees[treename].arrays(namedecode="utf-8"), **kwargs)
 
 
-def _import_uproot_that_can_write_root_files():
-    """uproot v4 can't handle writing files (ATOW),
-    so try it and fall back to uproot3
-    """
-    import uproot
-    try:
-        uproot.newtree
-    except AttributeError:
-        try:
-            import uproot3 as uproot
-        except ModuleNotFoundError as exc:  # pragma: no cover
-            exc.msg = (
-                "you have uproot {} installed, which does not support writing "
-                "ROOT files (yet), please install uproot3 "
-                "(see https://pypi.org/project/uproot3/)".format(
-                    uproot.__version__,
-                )
-            )
-            raise
-    return uproot
-
-
-def table_to_root(table, filename, treename="tree",
-                  overwrite=False, append=False, **kwargs):
+def table_to_root(
+        table,
+        filename,
+        treename="tree",
+        overwrite=False,
+        append=False,
+        **kwargs,
+):
     """Write a Table to a ROOT file
     """
-    uproot = _import_uproot_that_can_write_root_files()
-
-    createkw = {k: kwargs.pop(k) for k in {"compression", } if k in kwargs}
-    create_func = uproot.recreate if overwrite else uproot.create
-
-    if append is True:
-        raise NotImplementedError(
-            "uproot currently doesn't support appending to existing files",
-        )
+    import uproot
 
-    tree = uproot.newtree(dict(table.dtype.descr), **kwargs)
+    createkw = {
+        k: kwargs.pop(k) for k in {
+            "initial_directory_bytes",
+            "uuid_function",
+        } if k in kwargs
```

Comment:
Just checking that the change in the `kwargs` is not problematic. I don't know the details

Reply:
This is a necessary evil of this migration. I don't think these kwargs are ever used in the wild.

Reply:
Ok, thanks for confirming. I'll resolve this then